### PR TITLE
HV-1354 Make JavaFX classes visible when running the TCK in incontainer mode

### PR DESCRIPTION
**Pull request by gsmet**
* https://hibernate.atlassian.net/browse/HV-1354

### CODE REVIEW

File: tck-runner/src/test/modules/javafx/api/main/module.xml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                <path name="javafx/scene/media"/>
+                <path name="javafx/scene/paint"/>
+                <path name="javafx/scene/shape"/>
+                <path name="javafx/scene/text"/>
+                <path name="javafx/scene/transform"/>
+                <path name="javafx/scene/web"/>
+                <path name="javafx/stage"/>
+                <path name="javafx/util"/>
+                <path name="javafx/util/converter"/>
+                <path name="netscape/javascript"/>
```
Comment (gunnarmorling) [inline tck-runner/src/test/modules/javafx/api/main/module.xml]: Wow, we need that, too?

File: tck-runner/src/script/updateStandaloneXml.groovy [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    def text = file.text
+    file.write( processText( text ) )
+}
+
+// Add javafx.api module to the global modules
+standaloneXml = new File( project.properties['wildflyTargetDir'], 'standalone/configuration/standalone.xml' )
+println "[INFO] Add javafx.api as global module"
+
+processFileInplace( standaloneXml ) { text ->
+    text.replaceAll( /<subsystem xmlns="urn:jboss:domain:ee:4\.0">/, '<subsystem xmlns="urn:jboss:domain:ee:4.0">\n            <global-modules>\n                <module name="javafx.api" slot="main" />\n            </global-modules>' )
```
Comment (gunnarmorling) [inline tck-runner/src/script/updateStandaloneXml.groovy]: Nice!

File: tck-runner/src/test/modules/javafx/api/main/module.xml (hunk 1 above)
Comment (gsmet) [inline tck-runner/src/test/modules/javafx/api/main/module.xml]: For our usage, we don't. But I thought it was better to make the module as usable as possible.

**Comment by gunnarmorling** [comment]
Ok. All the com.sun.* stuff shouldn't be needed hopefully. Anyways, we
always can adapt this if needed.

**Pull request merged**